### PR TITLE
Don't enable excluders on Atomic

### PR DESCRIPTION
**Pull request by vrutkovs**
These cannot be installed there so these steps should be skipped on Atomic

### CODE REVIEW

**Comment by vrutkovs** [comment]
bot, retest this please

File: roles/openshift_excluder/tasks/exclude.yml [hunk 1]
```diff
@@ -1,28 +1,35 @@
...
+    when:
+    - r_openshift_excluder_enable_docker_excluder | bool
+    - docker_excluder_stat.stat.exists
 
-- name: Enable openshift excluder
-  command: "/sbin/{{ r_openshift_excluder_service_type }}-excluder exclude"
-  when:
-  - r_openshift_excluder_enable_openshift_excluder | bool
-  - openshift_excluder_stat.stat.exists
+  - name: Check for openshift excluder
```
Comment (mtnbikenc) [inline roles/openshift_excluder/tasks/exclude.yml]: Why is this task being run twice?

Comment (vrutkovs) [reply roles/openshift_excluder/tasks/exclude.yml]: Right, its not required. Fixed

**Pull request merged**